doc/send_receive_data.md

```diff
@@ -145,9 +145,10 @@ The application could combine `nb_bytes_ = 0` (no data to send now) and `fin = 1
 if finished sending that stream. The stack will then send a stream data frame with
 no content but the `FIN` bit set, marking the end of the stream.
 
-The application could also set `is_still_active=1`, or call `picoquic_mark_active_stream`
-from within the call back, which is equivalent. But that's a bit of a gamble.
-The stack will try to fill the current packet with some other content,
+The application could also set `is_still_active=1` from within the call back if more data is to be polled.
+It is to be noted that using mark_active_stream from within the callback is ambiguous, and may not behave the same way as `setting is_still_active=1`, since it is not linked to the callback context. 
```

Comment:
Suggestion: add an example, for clarity:

The application could also set `is_still_active=1` from within the call back if more data is to be polled, e.g.:
~~~
    /* Not sending here, waiting for a callback with a larger buffer */
    (void)picoquic_provide_stream_data_buffer(context, 0, 0, 1);
~~~

Reply:
I have added the example you suggested now as well